### ISSUE TITLE
Multi-line struct tags break syntax highlighting.

### ISSUE THREAD

**Issue by Splizard** [worlpaker/go-syntax#15]
If a file has a struct tag that spans multiple lines, syntax highlighting breaks.
<img width="633" alt="Screenshot 2024-06-05 at 9 17 02 AM" src="https://github.com/worlpaker/go-syntax/assets/2015791/ef136ef1-c7d5-4c24-a868-75fe7c92aaae">

I consider this a bug as this is a valid Go source file and  the Github highlighter handles this without issue:
```go
package main

type Struct struct {
	Value string `json:"value"
		this is not highlighted as a 'string'
	`
	AnotherValue string // all subsequent lines are highlighted as a string.
}
```

**Comment by worlpaker** [comment]
Thanks for the details! This bug should be fixed in v0.7.0.

**v0.7.0**
![example](https://github.com/worlpaker/go-syntax/assets/34960716/f58124f0-3c9a-4f79-9b8b-a4d026c564ac)

Can you please install the latest version of [go-syntax](https://marketplace.visualstudio.com/items?itemName=furkanozalp.go-syntax) and try it?

**Comment by worlpaker** [comment]
This bug has been fixed. Feel free to reopen it if you come across the same issue.
